.github/workflows/gempush.yml

```diff
@@ -0,0 +1,49 @@
+name: Ruby Gem
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
```

Comment:
I do not understand why it is done this way by all generators... I prefer to indend the `-` two spaces to the rig Iht. It reads much better to me that way. Like you have above on `on`:

```yaml
on:
  pull_request:
    branches:
      - master
  push:
    branches:
      - master
```